geonode/contrib/monitoring/models.py

```diff
@@ -1595,6 +1595,7 @@ def do_autoconfigure():
                 name=geonode[0],
                 url=geonode[1],
                 host=host,
+                last_check=datetime.now(),
```

Comment:
We must use 

from django.utils.timezone import now

in order to take timezone into account and not

datetime.now